pkg/features/kube_features.go

```diff
@@ -898,6 +898,13 @@ const (
 	// Enables user namespace support for stateless pods.
 	UserNamespacesSupport featuregate.Feature = "UserNamespacesSupport"
 
+	// owner: @mattcarry, @sunnylovestiramisu
+	// kep: https://kep.k8s.io/3751
+	// alpha: v1.29
+	//
+	// Enables user specified volume attributes for persistent volumes, like iops and throughput.
+	VolumeAttributesClass featuregate.Feature = "VolumeAttributesClass"
```

Comment:
If I understand correctly, the `VolumeAttributesClass` alpha API must also be enabled for this feature to be usable, but that this flag enables the functionality. Can we clarify that here to help prevent confusing when trying this out?

Reply:
@liggitt @msau42 Should we doing anything more than I suggested above to help cluster admins properly configure a feature that requires both a API be enabled and a feature gate be enabled?